mobile/src/modules/home/components/HomeStackedAreaChart.tsx

```diff
@@ -114,23 +127,31 @@ export class HomeStackedAreaChart extends React.PureComponent<TProps, State> {
     const now = new Date();
     const data: ChartData[] = [];
     const historicalPrice: Metric[] = [];
-    // tslint:disable-next-line: max-line-length
     // tslint:disable-next-line: one-variable-per-declaration
-    for (let d = new Date(new Date().setDate(now.getDate() - this.state.selectedDateRange)), i = 0;
-          d <= now;
-          d.setDate(d.getDate() + 1), i++) {
+    for (let d = new Date(new Date().setDate(
+        now.getDate() - this.state.selectedDateRange)
+      ), i = 0;
+        d <= now;
+        d.setDate(d.getDate() + 1), i++) {
       const atThatTime = d.getTime();
       data[i] = {
         day: new Date(d)
       };
-      if (this.state.priceInBTC) {
-        const price = this.props.priceApi.historicalPriceInfo.Data.find((metric) => {
-          return isSameDay(new Date(metric.time * 1000), d);
-        });
-        historicalPrice[atThatTime] = price || this.props.priceApi.historicalPriceInfo.Data[0];
+      if (this.props.priceApi.selectedCurrency === this.props.priceTypes[1] ||
```

Comment:
maybe invert it to `if it's not a BURST, then`

Reply:
good idea